Update frontmatter for devdot (#128)
hashicorp/terraform-docs-common: website/docs/registry/index.mdx

---
page_title: Terraform Registry Publishing
description: >-

>   The Terraform Registry is a public repository of Terraform providers, modules, and policies. 

Author: ```suggestion
  Publish Terraform providers, modules, and policies to the public Terraform Registry. 
```

>   The Terraform Registry is a public repository of Terraform integrations, modules, and policies. 

Author: ```suggestion
  The Terraform Registry is a public repository of Terraform providers, modules, and policies. 
```
